### DISCUSSION TITLE
Please go on with making the website

### DISCUSSION THREAD

**Discussion by TarunavBA** [TarunavBA/Social-Network#8] [Announcements]
Hey @dayvheed sorry for disturbing you but can you please go on with making the program? You can start with the Readme.md

**Comment by dayvheed** [comment]
Hi, @Tarunav BA ***@***.***>,

I definitely think we should have a meeting to discuss this. Maybe a Google
Meet meeting. Let me know what time works for you.

**Comment by TarunavBA** [comment]
Hey David but unfortunately I can't come for a meeting because í have to
attend my school too.. 😰 yea i am a 7th grade student. So if you need any
help you can create an issue with label help needed.

~ 🤖
